mesheryctl/internal/cli/root/system/error.go

```diff
@@ -42,6 +42,15 @@ const (
 	ErrInitPortForwardCode               = "1047"
 	ErrRunPortForwardCode                = "1048"
 	ErrFailedGetEphemeralPortCode        = "1049"
+	ErrProcessingConfigCode              = "1050"
+	ErrCreatingConfigFileCode            = "1051"
+	ErrAddingTokenToConfigCode           = "1052"
+	ErrAddingContextToConfigCode         = "1053"
+	ErrUnmarshallingConfigFileCode       = "1054"
+	ErrGettingRequestContextCode         = "1055"
+	ErrInvalidAPIResponseCode            = "1056"
+	ErrUnmarshallingAPIDataCode          = "1057"
+	ErrConnectingToServerCode            = "1058"
```

Comment:
you are supposed to wait for both pr's to merge, once theyre are merged you'll update your branch and call the error functions for returning errors . You still have to remove these functions @suhail34 pr and my  pr has some of them already implemented, its why i said u should look through the pr's ,compare them and remove what you have here to avoid duplicacy,  just ` return err` until the branches are merged . Then you can do the importing for returning the error functions/wrappers from the utils package. 
The ones that we didnt implement in our pr's you can define the and use them . 

Is this clear ?

Reply:
@Philip-21 Yeah, this make much sense.

Reply:
@Philip-21 @abdullah1308 I made changes, I think this is what you both want me to do. please check